### DISCUSSION TITLE
No Response From GitHub Support For Over 8 Weeks

### DISCUSSION THREAD

**Discussion by YouthfulRicker2** [community/community#137783] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I've had to create this account to even get an attempt at trying to recover my original account. 
I submitted a reinstatement ticket on JULY 18TH. (#2889914)
All my old repositories and projects are unaccessible and gone and it has been over 50 days. 
I am unable to do ANY work due to this, and haven't been able to for those past 8 weeks.

From what I remembered, github support used to be somewhat fast, like within the same month. but from what I've seen I can't rely on it anymore. What happened? Layoffs?

**Comment by SomethingNew71** [comment]
I am in the same boat, not quite as long as you but this is outrageous. If they plan to flag accounts to disable them in this way they need a fast avenue to resolve it after the issues have been fixed. I had an API key compromised and 5 issues created. Now my account and all its projects are held hostage and I cant deploy things for about a week now. No response on a ticket I opened.

**Comment by irvannasyakban** [comment]
same issue, My Ticket: [#2891688](https://support.github.com/ticket/2891688) don't get any respond from github support

**Comment by SomethingNew71** [comment]
Another week, no response at all from them and still a disabled account.

**Comment by SomethingNew71** [comment]
Now its been a full month with no response. I canceled my paid subscription until they fix this.

**Comment by baharbaba** [comment]
I've been waiting for a response for 1 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.
